PR_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+## JIRAEX-123 - Sample title
+#### JIRA: [jira link]
+#### Designs: [design link]
+
+## Description
+**Why is this change necessary?**
+* _what feature or bug were you addressing?_
+
+**What changes were made?**
+* _what specific changes were made?_
+
+**Potential side effects:**
+* _are there other parts of the code base that might be effected?_
+
+**Additional context or questions for reviewer (optional):**
+* _where should they start, related work, similar problem, etc._
+ 
+## Images/GIF
+
+SampleImage1.png
+
+## Testing
+- [ ] I have tested this work:
+
+   **By:**
+   1. [step 1]
+   2. [step 2]
+   3. [other steps]
+
+   **On:** 
+   - [ ] Chrome
+   - [ ] Safari
+   - [ ] Firefox
+   - [ ] IE
+   - [ ] Edge
+   - [ ] N/A
+
+## Checklist
```

Comment:
How would you feel about condensing these phrases? Ie. "Written tests" instead of "I have written tests that fail without this code."

Reply:
Yep, works for me.